dev-libs/angelscript/files/0004-Add-as_callfunc_arm64.cpp-to-gnuc-makefile.patch

```diff
@@ -0,0 +1,24 @@
+From 53841747c668abe3ff2cac280354703bef5dd395 Mon Sep 17 00:00:00 2001
```

Comment:
Can you include links to patch sources and any relevant bug links at the top of each patch? It saves digging in git history.

Reply:
I actually have no idea where this originally came from.  That line was included in the debian patch, in https://salsa.debian.org/yangfl-guest/angelscript/-/commits/master/debian/patches, but the timestamp does not make sense since arm64 did not exist in 2001 nor did angelscript.  I couldn't find anything on that commit hash.

The only interaction from the Debian maintainer is https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=997062 where they don't seem inclined to do anything with these.  There's no reference to where they originally came from in the Debian git history.

On the other hand I don't think they are entirely downstream patches, because all they do is add in code for the architecture which is already written and included in the sources.  It doesn't make sense for upstream to have written that code and just never built it.

I can link the Debian git sources in the patch, is that sufficient?

Reply:
Yeah, that'd be more than enough!